### PR TITLE
remove all `| table` from the book

### PR DESCRIPTION
**Pull request by amtoine**
related to 
- ~~https://github.com/nushell/nushell/pull/9633~~
- https://github.com/nushell/nushell/pull/9676
- https://github.com/nushell/nushell/issues/9630

with the `display_output` out of the way by default in https://github.com/nushell/nushell/pull/9633, we do not need anymore to pass big tables in `table` to make sure the hook does not trigger `table -e`

Commands used:
```nushell
sd ' ?\| table$' '' **/*
git restore commands/
```

### CODE REVIEW

File: book/pipelines.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ In effect, the command:
 And the pipeline:
 
 ```
-> ls | table
+> ls
```
Comment (jntrnr) [inline book/pipelines.md:82]: please don't change this one as it explains how `| table` is added for you automatically.

Comment (amtoine) [reply book/pipelines.md]: very good catch, thanks @jntrnr :scream: 

should be fixed in f3fd7eba :+1:

Comment (amtoine) [reply book/pipelines.md]: i've also added a note about `ls` and `ls | table` being *one and the same* in 17597f96, this is not true and caused more confusion in https://github.com/nushell/nushell/issues/9630 :thinking:

**Comment by amtoine** [comment]
now that nushell/nushell#9676 has landed, i think this is ready :yum:

**Pull request merged**